Sources/Remote Logging/Crash Logging/SentryEventJSException.swift

```diff
@@ -0,0 +1,98 @@
+import Foundation
+import Sentry
+
+public struct JSException {
+    public let type: String
+    public let value: String
+    public let stacktrace: [StacktraceLine]
+    public let context: [String: Any]
+    public let tags: [String: String]
+    public let isHandled: Bool
+    public let handledBy: String
+
+    public init(type: String, value: String, stacktrace: [StacktraceLine], context: [String : Any], tags: [String : String], isHandled: Bool, handledBy: String) {
+        self.type = type
+        self.value = value
+        self.stacktrace = stacktrace
+        self.context = context
+        self.tags = tags
+        self.isHandled = isHandled
+        self.handledBy = handledBy
+    }
+    
+    public struct StacktraceLine {
+        public let filename: String?
+        public let function: String?
+        public let lineno: NSNumber?
+        public let colno: NSNumber?
+        
+        public init(filename: String?, function: String?, lineno: NSNumber?, colno: NSNumber?) {
+            self.filename = filename
+            self.function = function
+            self.lineno = lineno
+            self.colno = colno
+        }
+    }
+}
+
+public class SentryEventJSException: Event {
+    required init() {
+        // All JavaScript exceptions should be trated as fatal errors
+        super.init(level: .fatal)
+        // Setting the event's platform to JavaScript is required by Sentry to be processed
+        // as a JavaScript exception. Otherwise, Sentry won't symbolicate the stack trace.
+        self.platform = "javascript"
+    }
+    
+    public static func initWithException(_ jsException: JSException) -> SentryEventJSException {
```

Comment:
Is it possible to declare this function as `public convenience init(exception: JSException)`?

Reply:
Suggestion applied in https://github.com/Automattic/Automattic-Tracks-iOS/pull/278/commits/ba6686962376f334881a18641515db92f0bfb4b0.